echemdb/data/cv/entry.py

```diff
@@ -85,7 +85,7 @@ def __getattr__(self, name):
 
             >>> entry = Entry.create_examples()[0]
             >>> entry.source
-            {'version': 1, 'doi': 'https://doi.org/10.1039/C0CP01001D', 'bib': 'alves_2011_electrochemistry_6010', 'figure': '2a', 'curve': 'solid'}
+            {'bib': 'alves_2011_electrochemistry_6010', 'curve': 'solid', 'doi': 'https://doi.org/10.1039/C0CP01001D', 'figure': '2a', 'version': 1}
```

Comment:
Should we rename `bib` to `key`, `citation key` or `identifier`?

Reply:
I agree. `bib` is not so useful. `identifier` sounds good to me, since it is used as such in the database. I will change that.

Reply:
Actually I just noticed that `identifier` is not such a good idea, since that is the name of the files. Here it is explicitly the bibliography key. So `citation key` it is.